integration-tests/test-api/README.md

```diff
@@ -1,8 +1,16 @@
 ```
-npm install
-npm run dev
+pnpm install
+pnpm run start
```

Comment:
These should be:

```
bun install
bun run start
```

Reply:
But imo we shouldn't encourage people to run the test-api locally, and just run it via docker.

Reply:
Sure, I'll remove these commands and leave the docker related ones I had added below it!